frontend/src/pages/chartTest.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Chart from 'chart.js';
+import Button from 'react-bootstrap/Button';
```

Comment:
The bootstrap in this file should be changed to MUI. Will be useful when you declare global MUI themes

Reply:
Addressed in afb14ddec0451920a81cdf939abbc84f9ba788be